db/migrate/20220714153418_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
```

Comment:
- [ ] Text type should be text not string. Kindly change it.

Reply:
- [ ] as the previous CR already mentioned please change the type for text to Text not string.